test/benchmark/shimmer/instrumentation.bench.js

```diff
@@ -40,12 +40,13 @@ suite.add({
   }
 })
 
-suite.add({
-  name: 'shimmer.reinstrument()',
-  agent: true,
-  fn: function (agent) {
-    return shimmer.reinstrument(agent, 'benchmark')
-  }
-})
+// This causes the cjs loader to fail, with benchmark not being found.
+// suite.add({
+//   name: 'shimmer.reinstrument()',
+//   agent: true,
+//   fn: function (agent) {
+//     return shimmer.reinstrument(agent, 'benchmark')
+//   }
+// })
```

Comment:
I was inclined to remove this block, but it may be possible to have this run with the right path, if we really wanted to run it for some reason.

Reply:
@jordigh The failure here is that `benchmark` can't be found from the context of the test, which is...weird. It's on my docket, along with the max-listeners-exceeded problem.